src/rust/src/incremental_merkle_tree.rs

```diff
@@ -181,6 +184,7 @@ pub fn write_bridge<H: HashSer + Ord, W: Write>(
 /// ordinary circumstances, the checkpoint ID will be the block height at which the checkpoint was
 /// created, but since we don't have any source for this information, we require the caller to
 /// provide it; any unique identifier will do so long as the identifiers are ordered correctly.
+#[allow(clippy::needless_borrows_for_generic_args)]
```

Comment:
It doesn't look to me as though there are any needless borrows in this method. Is this a false-positive that should be reported against clippy?